cluster/kubernetes/policies_test.go

```diff
@@ -99,7 +99,7 @@ func TestUpdatePolicies(t *testing.T) {
 	} {
 		caseIn := templToString(t, annotationsTemplate, c.in)
 		caseOut := templToString(t, annotationsTemplate, c.out)
-		out, err := UpdatePolicies([]byte(caseIn), c.update)
+		out, err := (&Cluster{}).UpdatePolicies([]byte(caseIn), c.update)
```

Comment:
Good evidence that we should split out `Cluster` and um, `ManifestRepo` (or something). But doesn't need to be done here and now.